### PR TITLE
Inanhkn/remove autoscaling movie

### PR DESCRIPTION
**Pull request by inanhkn**
@kimtonyhyun @forea 

In `view_ic_over_movie_interactively.m`, I scaled the movie roughly to `[0,1]` and set `CLim` parameters for `imagesc(...)` accordingly. Since every frame is now scaled by the same constant, traces should be more interpretable.

### CODE REVIEW

**Comment by kimtonyhyun** [comment]
Looks good to me, although we'll probably want to expose the `CLim` params at runtime (so the fixed scale can be adjusted on the fly).

File: classification/support/view_ic_over_movie_interactively.m [hunk 1]
```diff
@@ -10,15 +10,44 @@
...
+
+threshUpBright = quantile(maxVec,quant_upper);
+threshDownBright = quantile(maxVec,quant_lower);
+
+threshUpDark = quantile(minVec,quant_upper);
+threshDownDark = quantile(minVec,quant_lower);
+
+ZHigh = mean(maxVec(maxVec>threshDownBright & maxVec<threshUpBright));
+ZLow = mean(minVec(minVec>threshDownDark & minVec<threshUpDark));
+
```
Comment (kimtonyhyun) [inline classification/support/view_ic_over_movie_interactively.m]: Performing the above set of operations _for each IC_ is expensive and has considerably slowed down the load time of each plot.

**Comment by inanhkn** [comment]
I think I was a bit silly to put this in `view_ic_over_movie_interactively.m ` . This can be done once in `classify_ics.m` and the scaled movie can be passed to this script. We don't need to calculate `ZHigh, ZLow` for each IC.

**Comment by kimtonyhyun** [comment]
Yeah no worries, I have a fix for this,

-Tony

2015-02-08 14:28 GMT-08:00 inanhkn <notifications@github.com>:

**Pull request merged**